src/pfs/pfs.proto

```diff
@@ -191,6 +191,7 @@ message ListRepoRequest {
   // type is the type of (system) repos that should be returned
   // an empty string requests all repos
   string type = 1;
+  repeated string projects = 2;
```

Comment:
Should probably document that no projects means all projects, while some projects means just those projects.